ibis-server/justfile

```diff
@@ -31,8 +31,9 @@ run:
 dev:
     poetry run fastapi dev --port {{port}}
 
-test:
-    poetry run pytest
+# run the pytest with given args
+test *ARGS:
+    poetry run pytest {{ ARGS }}
```

Comment:
I prefer `poetry run pytest -m {{marker}}` to restrict contributors to using markers. If they want other testing methods, they can directly use the `poetry run pytest` command.

Reply:
I don't prefer this because `marker` would be the required option for running testing. However, if we just append the args to the tail, it's an optional args. `just test` also works for running all tests. It would like an alias of `poetry run pytest`.
If we want to have a command for restricting tests, I guess we can provider another just command for it. Maybe
```
db-test *ARGS:
    poetry run pytest -m {{ ARGS[0] }}
```
I'm not sure.

Reply:
We could only provide one command for testing the specific marker. We now have too many test suites requiring various Docker images, and the chances of needing to run all tests together are quite low. If there's a genuine need to run all tests together, one can execute `poetry run pytest` directly.